include/mamba/core/context.hpp

```diff
@@ -109,7 +109,15 @@ namespace mamba
 
         // TODO check writable and add other potential dirs
         std::vector<fs::path> envs_dirs = { root_prefix / "envs" };
-        std::vector<fs::path> pkgs_dirs = { root_prefix / "pkgs" };
+        std::vector<fs::path> pkgs_dirs
+            = { root_prefix / "pkgs",
+                fs::path("~") / ".mamba" / "pkgs"
+#ifdef _WIN32
+                ,
+                fs::path(std::getenv("APPDATA") ? std::getenv("APPDATA") : "") / ".mamba" / "pkgs"
```

Comment:
hm, i think there is a better way: https://github.com/mamba-org/mamba/blob/da87c2675370f2557369a982ac40aee809995e41/src/core/menuinst.cpp#L252-L253

Reply:
great thx

Reply:
do you know the name corresponding to appdata? this one is the userprofile

Reply:
I think we need to first factor out the code from `menuinst.cpp` and move it to some utils file.
Then we could use `https://docs.microsoft.com/en-us/windows/win32/shell/knownfolderid` on of the folderid's here (e.g. `FOLDERID_LocalAppData` or one of the others, not sure which one is the appropriate one).